shared/gh/partials/admin-batch-edit.html

```diff
@@ -32,8 +32,20 @@
                     </li>
                 </ul>
 
+                <!-- Series title -->
                 <% if (data.borrowedFrom) { %><i class="fa fa-link pull-left"></i> <% } %><h1 class="gh-jeditable-series-title"><%- data.series.displayName %></h1>
 
+                <!-- Borrowed data -->
+                <% if (data.borrowedFrom) { %>
+                    <p class="gh-series-title-borrowed-info">
+                        <% if(data.tripos) { %>
+                            <span>Belongs to: <a class="gh-series-borrowed-tripos" href="javascript:;" data-id="<%- data.tripos.id %>" title="<%- data.tripos.displayName %>"><%- data.tripos.displayName %></a></span>
```

Comment:
Do we have enough data to construct an actual link instead of updating the state programatically?